includes/pro/class.clef-pro.php

```diff
@@ -32,6 +32,10 @@ public function add_settings($form) {
     }
 
     public function ajax_get_pro_services() {
+        if (!$this->settings->is_configured()) {
```

Comment:
If the site is uninitialized, don't do a request to Clef, which was previously causing a 403.